scripts/compile_cost_assumptions.py

```diff
@@ -216,9 +216,44 @@ def get_data_DEA(tech, data_in):
     return df_final
 
 #
+
+def add_offwind_connection_data(costs):
+    """"
+    add connection costs for offshore wind from DEA
+    """
+    # station includes platform cost, project management and environmental
+    # assessment and transformer station cost
```

Comment:
maybe add comment, costs for offshore wind farms with distance > 10km from the coast (near shore wind farms are cheaper)